configs/orchestrator/wordle/1b.toml

```
@@ -0,0 +1,25 @@
+max_steps = 500
+batch_size = 1024
+micro_batch_size = 2
```

```suggestion
micro_batch_size = 1
```
maybe this should remove the need for activation checkpointing ?

---

reverted + removed AC